ui/app/routes/vault/cluster/meep.js

```diff
@@ -0,0 +1,8 @@
+import Route from '@ember/routing/route';
+import ClusterRoute from 'vault/mixins/cluster-route';
+
+export default Route.extend(ClusterRoute, {
+  model() {
+    // return this.store.queryRecord('license', {});
```

Comment:
Here's a link on the significance of a route's model.  https://guides.emberjs.com/v3.8.0/routing/specifying-a-routes-model/